.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+:memo: :lipstick: :new: :bomb: :bug: :art: Replace this line with relevant emojis
```

Comment:
May want to say to include them in the title as well. And maybe the description of what they are used for (so, link to CONTRIBUTING probably).